lib/protobuf/generators/enum_generator.rb

```diff
@@ -32,6 +32,7 @@ def compile
 
       def build_value(enum_value_descriptor)
         name = enum_value_descriptor.name
+        name.upcase! if ENV.fetch('PB_ENUM_UPCASE', '') != ''
```

Comment:
what about `ENV['PB_ENUM_UPCASE'].present?` (activesupport is already a dependency of this gem, so we could just add `require 'activesupport/core_ext/object/blank'`)